builtin/providers/aws/resource_aws_s3_bucket.go

```diff
@@ -1290,6 +1334,19 @@ func normalizeRegion(region string) string {
 	return region
 }
 
+func validateS3BucketAccelerationStatus(v interface{}, k string) (ws []string, errors []error) {
+	value := strings.ToLower(v.(string))
+	validTypes := map[string]struct{}{
+		"Enabled":   struct{}{},
+		"Suspended": struct{}{},
```

Comment:
Whoops! I need to admit that I tried to be really clever here and add a unit test after I created the PR that would check for the types :(

Good spot!!!

Fixed now